Header header
geometry_msgs/Pose pose

int32 K4ABT_JOINT_PELVIS=0
int32 K4ABT_JOINT_SPINE_NAVAL=1
int32 K4ABT_JOINT_SPINE_CHEST=2
int32 K4ABT_JOINT_NECK=3
int32 K4ABT_JOINT_CLAVICLE_LEFT=4
int32 K4ABT_JOINT_SHOULDER_LEFT=5
int32 K4ABT_JOINT_ELBOW_LEFT=6
int32 K4ABT_JOINT_WRIST_LEFT=7
int32 K4ABT_JOINT_CLAVICLE_RIGHT=8
int32 K4ABT_JOINT_SHOULDER_RIGHT=9
int32 K4ABT_JOINT_ELBOW_RIGHT=10
int32 K4ABT_JOINT_WRIST_RIGHT=11
int32 K4ABT_JOINT_HIP_LEFT=12
int32 K4ABT_JOINT_KNEE_LEFT=13
int32 K4ABT_JOINT_ANKLE_LEFT=14
int32 K4ABT_JOINT_FOOT_LEFT=15
int32 K4ABT_JOINT_HIP_RIGHT=16
int32 K4ABT_JOINT_KNEE_RIGHT=17
int32 K4ABT_JOINT_ANKLE_RIGHT=18
int32 K4ABT_JOINT_FOOT_RIGHT=19
int32 K4ABT_JOINT_HEAD=20
int32 K4ABT_JOINT_NOSE=21
int32 K4ABT_JOINT_EYE_LEFT=22
int32 K4ABT_JOINT_EAR_LEFT=23
int32 K4ABT_JOINT_EYE_RIGHT=24
int32 K4ABT_JOINT_EAR_RIGHT=25
int32 K4ABT_JOINT_COUNT=26
